tests/mainnet.rs

```diff
@@ -53,20 +57,30 @@ fn mainnet_blocks_from_rpc() {
             fs::create_dir_all(dir.clone()).unwrap();
             let file_block = File::create(format!("{dir}/block.json")).unwrap();
             serde_json::to_writer(file_block, &block).unwrap();
-            let file_state = File::create(format!("{dir}/state_for_execution.json")).unwrap();
-            serde_json::to_writer(file_state, &rpc_storage.get_cache()).unwrap();
+
+            let accounts: BTreeMap<Address, PlainAccount> =
```

Comment:
I'd add a quick comment:
```
// We convert to `BTreeMap`s for consistent ordering & diffs
```

Reply:
Oh wait, it's not working as intended yet. More information: https://github.com/serde-rs/json/issues/1000#issuecomment-1464980111.

Reply:
let's postpone this issue for now